Kiosk/Views/InsuranceClaimAutomation/DetailsViewControl.xaml

```diff
@@ -0,0 +1,112 @@
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+
+        <Button Grid.RowSpan="2" Click="OnCloseDetailsViewButtonClicked" Width="40" Height="40" Background="{StaticResource CloseButtonBackgroundColor}" VerticalAlignment="Top" HorizontalAlignment="Right">
+            <SymbolIcon Symbol="Cancel"/>
+        </Button>
+
+        <TextBlock Text="Claim detail" Style="{StaticResource SubtitleTextBlockStyle}" Margin="24,20,24,0" Opacity="0.8"/>
+
+        <Grid Grid.Row="1" Margin="24" RowSpacing="30">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="*"/>
+            </Grid.RowDefinitions>
+
+            <StackPanel Spacing="18">
+                <StackPanel.Resources>
+                    <Style TargetType="StackPanel">
+                        <Setter Property="Spacing" Value="4"/>
+                    </Style>
+                    <Style TargetType="TextBlock">
+                        <Setter Property="Opacity" Value="0.8"/>
+                    </Style>
+                </StackPanel.Resources>
+
+                <StackPanel>
+                    <TextBlock Text="Claim ID" Style="{StaticResource HeaderTextBlock}"/>
+                    <TextBlock x:Name="claimIdTextBlock"/>
+                </StackPanel>
+                <StackPanel>
+                    <TextBlock Text="Customer name" Style="{StaticResource HeaderTextBlock}"/>
+                    <TextBlock x:Name="customerNameTextBlock"/>
+                </StackPanel>
+                <StackPanel>
+                    <TextBlock Text="Date" Style="{StaticResource HeaderTextBlock}"/>
+                    <TextBlock x:Name="dateTextBlock"/>
+                </StackPanel>
+                <StackPanel>
+                    <TextBlock Text="Warranty ID" Style="{StaticResource HeaderTextBlock}"/>
+                    <TextBlock x:Name="warrantyIdTextBlock" />
+                </StackPanel>
+                <StackPanel>
+                    <TextBlock Text="Warranty amount" Style="{StaticResource HeaderTextBlock}"/>
+                    <TextBlock x:Name="warrantyAmountTextBlock"/>
+                </StackPanel>
+                <StackPanel>
+                    <TextBlock Text="Invoice total" Style="{StaticResource HeaderTextBlock}"/>
+                    <TextBlock x:Name="totalTextBlock"/>
+                </StackPanel>
+            </StackPanel>
+
+            <Pivot Grid.Row="1" Background="{StaticResource PivotBackgroundColor}">
+                <PivotItem Margin="17,13,17,19">
+                    <PivotItem.Header>
+                        <TextBlock Text="Product image" FontSize="15" />
+                    </PivotItem.Header>
+
+                    <Grid Background="{StaticResource PivotItemBackgroundColor}">
+                        <Grid HorizontalAlignment="Center">
+                            <ctl:ImageWithFaceBorderUserControl x:Name="productImageControl" />
```

Comment:
Let's just use a Image control for this.

Reply:
Fixed.